src/current/_includes/releases/v24.3/v24.3.2.md

```diff
@@ -0,0 +1,18 @@
+## v24.3.2
+
+Release Date: December 26, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v24-3-2-sql-language-changes">SQL language changes</h3>
```

Comment:
this is the fix that's being introduced with the v24.3.2 (hot fix). from the 24.3 backport https://github.com/cockroachdb/cockroach/pull/137919:

> Release note (sql change): The legacy_varchar_typing session setting
has been added, which reverts the changes of https://github.com/cockroachdb/cockroach/pull/133037 that cause the
change in typing behavior described in https://github.com/cockroachdb/cockroach/issues/137837.